content/applications/services/helpdesk/overview/receiving_tickets.rst

```diff
@@ -4,274 +4,338 @@ Start receiving tickets
 
 Odoo *Helpdesk* offers multiple channels where customers can reach out for assistance, such as
```

Comment:
I can't leave a note directly on the title of this doc, but I'd suggest changing it to "Helpdesk tickets" or something similar to that, to avoid starting the doc title with a verb, as we've been instructed to avoid. I know it's a pre-existing title and you totally didn't name it that, but figured it was worth mentioning, as I just noticed it. 👍

Reply:
I don't necessarily want to change it to specifically "Helpdesk tickets" because it is about the ways they are received, not the tickets themselves. I confirmed in the content guidelines that verbs are ok, so I'd like to keep that, but I will change it to "Receiving tickets" and remove "start" 
https://www.odoo.com/documentation/17.0/contributing/documentation/content_guidelines.html#titles-and-headings